source/_includes/navigation.html

```diff
@@ -28,7 +28,7 @@
 		<li>
 			<a href="#">Posts</a>
 			<ul class="dl-submenu">
-				<li><a href="/posts/">All Posts</a></li>
+				<li><a href="/archives/">Archives</a></li>
```

Comment:
Is it really necessary to have both, if they're doing the same thing? I believe the concept of "Archives" to have more appeal to the reader than "All Posts" because it suggests the posts are categorized and organized in some way.

I understand that some may like "/posts/" better and that it may be prefered by some SEO techniques, but I don't see any reason to have both of them behaving the same way. If an author wants to rename this index, he could just rename the file.